go.mod

```diff
@@ -1,6 +1,6 @@
-module github.com/buildkite/go-buildkite/v2
+module github.com/buildkite/go-buildkite/v3
 
-go 1.15
+go 1.16
```

Comment:
This bump [drops support for macOS 10.11](https://github.com/buildkite/agent/blob/main/docs/macos.md) which is likely fine I just wanted to note it 😄 we don’t support 10.11 in the agent any more either

Reply:
Good to know!